liblouis/compileTranslationTable.c

```diff
@@ -1127,7 +1127,7 @@ addCharacterClass(const FileInfo *file, const widechar *name, int length,
 				// don't abort because in some cases (before/after rules)
 				// this will work fine, but it will not work in multipass
 				// expressions
```

Comment:
The comment will have to be change/deleted too.

Reply:
So:
This comment already not need, because base opcodes works already right with multipass rules? Safe I remove this comment part with you linked the review?
Or, if the comment is need, but need rewrite, to help me have possibility to you write a correct english comment with good the changed code part?

Reply:
I just removed the comment